test/unit/reductions/exponentiated_gradient/test_exponentiatedgradient_smoke.py

```diff
@@ -103,7 +109,7 @@ def test_smoke(self, testdata):
 
     def test_simple_fit_predict(self):
         estimator = LeastSquaresBinaryClassifierLearner()
-        constraints = DemographicParity()
+        constraints = EqualOpportunity()
```

Comment:
You may add another test case, but please leave the existing ones in place :-)

Reply:
Sorry about this. Yes, I will do this.